### PR TITLE
Timeline layout

### PR DESCRIPTION
**Pull request by fenglish**

### CODE REVIEW

File: scss/layout/_timeline.scss [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+			padding-top: 10px;
+			border-bottom: 0px;
+			@include oTypographySansBold($scale: 8, $progressive: true);
+		}
+		.n-content-layout__slot {
+			flex-basis: 100%;
+			padding-left: 30px;
+			margin-left: 0px;
+			margin-top: 0px;
+			margin-bottom: 0px;
```
Comment (florianherrengt) [inline scss/layout/_timeline.scss]: can this be a one line instead of many margin?

File: scss/layout/_timeline.scss [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+			position: relative;
+			&:before {
+				content: '';
+				width: 2px;
+				background: #000;
```
Comment (florianherrengt) [inline scss/layout/_timeline.scss]: I think we're should use sass `getColor`

**Comment by jonnywyatt** [comment]
Not sure if it's within the scope of this PR, and I didn't want to nitpick in slack, but the posted example at https://www.ft.com/content/8b502116-9205-11e5-94e6-c5413829caa5 has h2's for months nested inside h2's for years. Is that a CMS content issue?

**Comment by fenglish** [comment]
@jonnywyatt I don't know the rule to change xml into html. Looks like they set h2 purposely.
https://github.com/Financial-Times/next-es-interface/blob/master/server/stylesheets/headings.xsl
We should ask @adgad when he is back.

**Pull request merged**